### PR TITLE
Add ETHLatam @ Buenos Aires to upcoming events

### PR DESCRIPTION
**Pull request by LichuAcu**
## Description

I added ETHLatam @ Buenos Aires to the list of upcoming events 📅
https://twitter.com/ethlatam

### CODE REVIEW

**Comment by wackerow** [comment]
Hey @LichuAcu, thanks for the post...
image
Is the link here correct? Anyone else getting this?
If we can double-check/correct we should be able to get this in soon =)

**Comment by pettinarip** [comment]
@LichuAcu wohooo! see you there! (wen tickets 🤔)

@wackerow Yes, I think the website is not online yet. Perhaps we can set the twitter account in the meantime https://twitter.com/ethlatam no?

File: src/data/community-events.json [hunk 1]
```diff
@@ -269,6 +269,15 @@
     "startDate": "2022-07-19",
     "endDate": "2022-07-21"
   },
+  {
+    "title": "ETH Latam @ Buenos Aires",
+    "to": "https://ethlatam.org/",
```
Comment (wackerow) [inline src/data/community-events.json:274]:
```suggestion
    "to": "https://twitter.com/ethlatam",
```

File: src/data/community-events.json [hunk 2]
```diff
@@ -269,6 +269,15 @@
     "startDate": "2022-07-19",
     "endDate": "2022-07-21"
   },
+  {
+    "title": "ETH Latam @ Buenos Aires",
+    "to": "https://twitter.com/ethlatam",
+    "sponsor": null,
+    "location": "Buenos Aires, Argentina",
+    "description": "A 4-day conference that builds upon the work of the local pioneers that laid the foundations for a vibrant community that made crypto mainstream in Argentina, one of the countries where it’s more needed. Join us for the very first edition of the event.",
```
Comment (minimalsm) [inline src/data/community-events.json:277]:
```suggestion
    "description": "A 4-day conference that builds upon the work of the local pioneers that laid the foundations for a vibrant community that made crypto mainstream in Argentina.",
```

File: src/data/community-events.json [hunk 3]
```diff
@@ -269,6 +269,15 @@
     "startDate": "2022-07-19",
     "endDate": "2022-07-21"
   },
+  {
+    "title": "ETH Latam @ Buenos Aires",
```
Comment (minimalsm) [inline src/data/community-events.json:273]:
```suggestion
    "title": "ETHLatam",
```

**Review by minimalsm — APPROVED** [review]
Hey @LichuAcu, thanks for the PR!

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And we've just released our 2022 POAPs so remember to [claim yours](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by minimalsm** [comment]
@all-contributors please add @LichuAcu for docs

**Pull request merged**